lib/Conversion/ExportVerilog/PrepareForEmission.cpp

```diff
@@ -49,7 +49,7 @@ bool ExportVerilog::isSimpleReadOrPort(Value v) {
   auto readSrc = read.getInput().getDefiningOp();
   if (!readSrc)
     return false;
-  return isa<WireOp, RegOp>(readSrc);
+  return isa<WireOp, RegOp, LogicOp, XMROp>(readSrc);
```

Comment:
The addition of `LogicOp` seems unrelated to this PR?

Reply:
Right, unrelated (I found I was missing LogicOp either :). I'll land separately.